src/debuginfo.cpp

```diff
@@ -330,11 +329,11 @@ class JuliaJITEventListener: public JITEventListener
             if (!Addr) continue;
 #   endif
 #elif defined(_OS_WINDOWS_)
-#   if defined(LLVM36)
-            SectionSize = Section->getSize();
+#   if defined(LLVM37)
+            assert(obj.isELF)
```

Comment:
this assert seems to be incorrect. COFF also has this method (http://llvm.org/docs/doxygen/html/COFFObjectFile_8cpp_source.html#l00239)